ext/spl/spl_directory.h

```diff
@@ -105,6 +105,7 @@ struct _spl_filesystem_object {
 #define SPL_FILE_DIR_KEY_AS_PATHNAME       0x00000000 /* make RecursiveDirectoryTree::key() return getPathname() */
 #define SPL_FILE_DIR_KEY_AS_FILENAME       0x00000100 /* make RecursiveDirectoryTree::key() return getFilename() */
 #define SPL_FILE_DIR_KEY_MODE_MASK         0x00000F00 /* mask RecursiveDirectoryTree::key() */
+#define SPL_FILE_NEW_CURRENT_AND_KEY       SPL_FILE_DIR_KEY_AS_FILENAME|SPL_FILE_DIR_CURRENT_AS_FILEINFO
```

Comment:
Does this even need to be defined? Wouldn't it be possible to compose the constant values in the stub? (or am I asking for a feature which does not yet exist)

Reply:
Yes, it's not strictly required to have, but I prefer not to put too much C code into stubs, so I usually create a new macro when an expression needs to be used.